gcp/deployment-packages/autoscale-byol/README.md

```diff
@@ -0,0 +1,130 @@
+# GCP Deployment Manager package for Check Point Autoscaling BYOL solution
+This directory contains CloudGuard IaaS deployment package for Check Point Autoscaling (BYOL) solution published in the [GCP Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/checkpoint.vsec?tab=Overview).
+
+# How to deploy the package manually
+To deploy the Deployment Manager's package manually, without using the GCP Marketplace, follow these instructions:
+1. [Download](https://github.com/CheckPointSW/CloudGuardIaaS/archive/master.zip) the CloudGuardIaaS repository
```

Comment:
Change to:
1. Clone or download the files in this directory

Reply:
Fixed